client/js/gdpr/chart.js

```diff
@@ -275,7 +275,7 @@ export class GdprChart extends HTMLElement {
         }));
 
     const pointsStr = points
-        .map(({x, y}) => `${x}, ${y}`)
+        .map(({x, y}) => `${x * this.zoom}, ${y}`)
```

Comment:
You should apply your zoom earlier when you apply RUN_SCALE_X to the points. This will keep you from needing to re-apply zoom below on your start and end caps.